snippets/rmarkdown.json

```diff
@@ -67,5 +67,23 @@
 		"prefix": "image",
 		"body": "![${1:alt}](http://${2:link})$0",
 		"description": "Insert image"
+	},
+	"Insert code chunk": {
```

Comment:
Duplicated with line 22 "Insert fenced code block" on the original snippet.
Also, I suggest that can follow original names such as
"Insert R code chunk" ->"Insert R fenced code block"

Reply:
"Fenced code block" is more of a term in Markdown.
"Code chunk" is more of a term in R Markdown (as widely used in R markdown documentations e.g https://rmarkdown.rstudio.com/lesson-3.html, https://bookdown.org/yihui/rmarkdown/r-code.html) and is used in RStudio.

I suggest that we follow the "official names" of such and R markdown users might be more familiar with them?

Reply:
From my understanding, code chunk in R Markdown specifically means something like

````
```{language}

```
````

for rmarkdown renderer to use while fenced code block means

````
```language

```
````

where `language` is not enclosed by ``{}`.

Reply:
To be more clear, what about 

`chunk` -> `code chunk`
`r chunk` -> `r code chunk`

Reply:
> I suggest that we follow the "official names" of such and R markdown users might be more familiar with them?

Thank you for your knowledge, I did not know R markdown standard names, 

your suggestion
> chunk -> code chunk
> r chunk -> r code chunk

Looks good that looks easy for both RStudio user and vs code user.